tool/src/main/java/org/wildfly/security/tool/ElytronToolMessages.java

```diff
@@ -422,6 +451,24 @@
             "Blocks of options must be separated by a blank line.")
     String cmdFileSystemRealmBulkConvertDesc();
 
+    @Message(id = NONE, value = "Bulk conversion with options listed in description file. Optional options have default values, required options do not. (Action) %n" +
+            "The options realm-name, hash-charset, hash-encoding, and levels are optional. %n" +
+            "The options credential-store and secret-key are also both optional, but they must either both be unspecified or both be specified. %n" +
```

Comment:
What happens if `credential-store` and `secret-key` aren't specified? What does the command do in that case?

Reply:
I see the answer below. :)

Reply:
Is there a reason to default to "mycredstore.cs"? 
In your wildfly-core changes, `credential-store` and `secret-key` don't have default values right? I don't think we need to set these defaults here.

Reply:
The intention is that if they aren't specified, during the realm conversion (to encrypt it) it will make a new credential store with the name `mycredstore.cs` and a secret-key alias of `key` for use when encrypting the old realm.

Reply:
Thanks! In other Elytron Tool commands, do we automatically generate params like this?

Reply:
I don't think we should assume a filename in case this filename already exists on their filesystem.

Instead, as mentioned below, I think we should introduce a "create" parameter that indicates whether or not the credential store should be created if the provided path doesn't exist.

Reply:
Added create parameter